src/clusterfuzz/_internal/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -27,8 +27,7 @@ def linkify_kernel_or_lkl_stacktrace_if_needed(crash_info):
   """Linkify Android Kernel or lkl stacktrace."""
   kernel_prefix = ''
   kernel_hash = ''
-  if (environment.is_android_kernel() and
```

Comment:
I think this change is from a revert but @marktefftech do you know what it should be now?